### ISSUE TITLE
bootstrap /*! important comment */ causes missing rules

### ISSUE THREAD

**Issue by techieshark** [css/csso#251]
Some framework stylesheets (Bootstrap, HTML5 Boilerplate, normalize.css, etc) often include [special comments](http://yui.github.io/yuicompressor/css.html#special-comments) with source or license information. For example, Bootstrap has a number of CSS rules, followed by: 
````
/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */
````
followed by more CSS rules.

When CSSO processes this, some structural optimizations can fail. 

For example, here's a simple test file I passed through csso version 1.4.1:
````
h1 {
    font-size: 2em;
}
/*! foo */
h1 {
    font-size: 36px
}
````

The second `h1`'s `font-size` should override the first, so you'd expect to see:
````
/*! foo */
h1{font-size:36px}
````
in the output. Instead, here's what we actually see. 
````
h1{font-size:2em}
/*! foo */
````

I'm curious if issue #248 could be related.

**Comment by lahmatiy** [comment]
You're right. Issue #248 is connected with this one.
For now blocks between `/*! ... */` comment are compressing as separate files. Otherwise content of blocks will be mixed and license comments have no sense.
I found problem and going to fix it.

**Comment by lahmatiy** [comment]
Done.
Your example 

```css
h1 {
    font-size: 2em;
}
/*! foo */
h1 {
    font-size: 36px
}
```

Is now compressing to 

```css
h1{font-size:2em}
/*! foo */
h1{font-size:36px}
```

In future overridden rulesets will be removed, but content should not to be mixed still. Probably there will be an option that allows content mixing and grouping license comments.

**Comment by DennisBecker** [comment]
Will there be an option to also remove those comments and then merge those blocks? Or is this an adopted standard to mark copyrights?

**Comment by lahmatiy** [comment]
@DennisBecker Yep, this is an adopted standard to mark copyrights. Option you propose is possible, but I think doing that isn't correct.

**Comment by DennisBecker** [comment]
I'm not sure if the copyright has a meaning when you might have changed the code. If CSSO doesn't touch the definition and does not optimize the blocks then it is a must to include the copyright.

**Comment by lahmatiy** [comment]
@techieshark Thank you for your report. Problem solved, try (1.4.2)[https://github.com/css/csso/releases/tag/v1.4.2]

**Comment by techieshark** [comment]
@lahmatiy works great, thanks for the quick fix! :+1: :+1:
